Orange/widgets/evaluate/owtestlearners.py

```diff
@@ -326,11 +326,12 @@ def update_progress(finished):
         elif self.resampling == OWTestLearners.LeaveOneOut:
             results = Orange.evaluation.LeaveOneOut(
                 self.data, learners, **common_args)
-        elif self.resampling == OWTestLearners.Bootstrap:
-            results = Orange.evaluation.Bootstrap(
+        elif self.resampling == OWTestLearners.ShuffleSplit:
+            train_size = self.sample_p / 100
+            results = Orange.evaluation.ShuffleSplit(
                 self.data, learners, n_resamples=self.n_repeat,
-                p=self.sample_p / 100, random_state=rstate,
-                **common_args)
+                train_size=train_size, test_size=1 - train_size,
```

Comment:
This sometimes produces an error saying that test and train data sizes sum to num_instances + 1. Which should probably be fixed in scikit-learn, but in the meantime using the following works and is nicer anyway:
`test_size=None`